dapp/ledger-manifest-testnet-preview.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "acre-testnet",
+  "name": "Acre [testnet - preview]",
+  "url": "https://deploy-preview-708--acre-dapp-testnet.netlify.app/?embed=ledger-live",
```

Comment:
We configured netlify branch deployments for this feature branch, so we can use `https://ledger-live-updates--acre-dapp-testnet.netlify.app/?embed=ledger-live` here.